server/lib/app_manager.coffee

```diff
@@ -59,6 +63,14 @@ class AppManager
                     state: data.app.state
                 callback null, data.app.port
 
+    # readfile to start a static app
+    startStaticApp: (appName, url, path, callback) ->
+        logger.info "Starting static app #{appName}"
+        url += 'index.html' if url is '/' or url is '/public/'
+        fs.readFile path + url, 'utf8', (err, content) ->
```

Comment:
I think that I've managed to block the root path (if a user has the bad idea to link his app by doing href="../../../blog.html" for example). However i tested this situation and the answer of the proxy server is a 404 not found which sounds like logical for me. So i can't really see the use of this yet. Maybe there's something i did'nt consider. If anyone can help me with this particularity that would be great.